plot.go

```diff
@@ -106,8 +106,10 @@ func New() (*Plot, error) {
 		Legend:          legend,
 	}
 	p.Title.TextStyle = draw.TextStyle{
-		Color: color.Black,
-		Font:  titleFont,
+		Color:  color.Black,
+		Font:   titleFont,
+		XAlign: -0.5,
```

Comment:
I was always pretty happy with this XAlign and YAlign design for aligning text, but reading it years later, it's pretty opaque. Maybe we could make some helpful, and more-readable functions to set it:

```
func CenteredAbove(sty *TextStyle) {
    sty.XAlign = -0.5
    sty.YAlign = -1
}
// etc.
```